src/utils/dom.js

```diff
@@ -86,6 +86,7 @@ const sweetHTML = `
    <div class="${swalClasses.icon} ${iconTypes.error}">
      <span class="swal2-x-mark"><span class="swal2-x-mark-line-left"></span><span class="swal2-x-mark-line-right"></span></span>
    </div>
+   <h2 class="${swalClasses.titleFirst}" id="${swalClasses.titleFirst}"></h2>
```

Comment:
I don't really like that solution :/ It would be better to invert elements programmatically or use CSS.